### ISSUE TITLE
[BUG] Some Required Theme Snaps are missing.

### ISSUE THREAD

**Issue by Sabyasachi-Seal** [vinceliuice/Orchis-theme#286]
I keep getting this notification saying that some required theme snaps are missing.
Seems like this is a common issue with GTK and Orchis. Any Fixes?

**Comment by jpjarnoux** [comment]
Same issue

**Comment by jpjarnoux** [comment]
Okay,
I could see there is Flatpak now ?
Which way is the best for you ?
Thanks

**Comment by Sabyasachi-Seal** [comment]
I installed with the normal installation procedure with install.sh, but this is a notification that comes up every time I boot , not present with other themes.

**Comment by vinceliuice** [comment]
What's your system? I did not add any notification in my theme installation

**Comment by Sabyasachi-Seal** [comment]
Its Ubuntu 22.04

**Comment by vinceliuice** [comment]
I think it's your system notification to tell you install some snap version themes for your deskktop you can just click it to install  them

**Comment by Sabyasachi-Seal** [comment]
Checked again. I uninstalled Orchis and the notification no longer appears.
There is a related issue in WhiteSur GTK theme: [here](https://github.com/vinceliuice/WhiteSur-gtk-theme/issues/386)

**Comment by 4sa617** [comment]
I just installed the Orchis theme myself and I am getting the pop up. Ubuntu 22.04 as well.

**Comment by tranzzik** [comment]
Yep same issue here on Orchis, trying to install them just results in the same failed message.

**Comment by fregie** [comment]
Same issue too

**Comment by etothex23** [comment]
With Ubu budgie 10.6.4 linux 5.19.0 after installing orchis from snap I get this message.

**Comment by viniasbr** [comment]
Installed with install.sh and I get the same error message on Ubuntu 22.04

**Comment by jtrosper** [comment]
Fresh install of Ubuntu Budgie 22.04. Same error message. Clicking the notification does nothing.

**Comment by cchaires** [comment]
Same here with Ubuntu 23.04 Gnome 44 I use these options: 
`./install.sh -d /usr/share/themes/  -t green -c dark -s compact -l --tweaks compact --shell 44`

**Comment by joejoejar** [comment]
same issue Ubuntu 22.04

**Comment by vinceliuice** [comment]
1.  Uninstall `gtk-common-themes` using: `sudo snap remove gtk-common-themes`
2. Reinstall it again with: `sudo snap install gtk-common-themes --channel=latest/candidate` (`latest/beta` and `latest/edge` should also work)
3.  reboot your session/re-login

**Comment by Sabyasachi-Seal** [comment]
@vinceliuice tried. Didn't fix it

**Comment by yudistirabagus** [comment]
> @vinceliuice tried. Didn't fix it

same

**Comment by thattemperature** [comment]
same issue Ubuntu 22.04

**Comment by lxe** [comment]
Bumping this. Orchis with Ubuntu 23.10

**Comment by BlackWiCKED** [comment]
Ended up uninstalling for now, but happy to test again if there is a workaround.

**Comment by iskizreal** [comment]
Has anyone fixed this? The same error on ubuntu 22.04

**Comment by abbasnaqdi** [comment]
Same issue

**Comment by kwiza12** [comment]
Same issue in Ubuntu 22.04

**Comment by kauanmocelin** [comment]
Same here on ubuntu 24.04 installed manually using ./install.sh, very annoying notification.

**Comment by hemangjoshi37a** [comment]
same issue here. have anyone found any solution yet for this ?

**Comment by H0lz3r-x64** [comment]
same here! >:(

**Comment by fildunsky** [comment]
Yep, annoying issue. I have to remove this theme, unfortunately...

**Comment by stgpepper** [comment]
Ubuntu 24.04 and same popup issue with purple pack both pre-"compiled" and self installed via script tested. styles changed slightly, but I can see most of the styles were not applied overall.

> I think it's your system notification to tell you install some snap version themes for your deskktop

Can we track down what application or process is prompting this? Perhaps is one of the dependencies of Orchis that is actually causing it to appear? If so this question could be redirected to that specific project.

**Comment by pixildar** [comment]
ubuntu 24.10 same issue

**Comment by rovelcio** [comment]
> [vinceliuice on Jul 29, 2023](https://github.com/vinceliuice/Orchis-theme/issues/286#issuecomment-1656633875)
> 1. Uninstall `gtk-common-themes` using: `sudo snap remove gtk-common-themes`
> 2. Reinstall it again with: `sudo snap install gtk-common-themes --channel=latest/candidate` (`latest/beta` and `latest/edge` should also work)
> 3. reboot your session/re-login

After some attempts, I managed to make the error go away on a fresh install of Ubuntu 24.04 following @vinceliuice instructions (with some extra/different steps):

1. Go to gnome tweaks "Appearance" tab and set "Shell" and "Legacy Applications" to some other theme that is NOT Orchis, and preferably built-in (Yaru in my case, but any other should be fine)
2. Log out of your session and then log back in
3. Go into the folder you downloaded Orchis into and run `./install.sh -u` to uninstall the theme
4. Run `sudo snap remove gtk-common-themes` and then `sudo snap install gtk-common-themes --channel=latest/beta` (I tried the other versions, only latest/beta worked for me)
5. Still in the folder you downloaded Orchis into, run `./install.sh` with the options of your preference (mine were `./install.sh -d ~/.themes -t teal -c dark --tweaks macos dock`).
6. Go back into gnome tweaks and select Orchis on "Shell" and "Legacy Applications".

I'm not sure if all the steps are required and if their order matters, but there you go.

**Comment by sinnbeck** [comment]
@rovelcio Just tried this on a brand new install but sadly I am still getting the error on each login :(

Has anyone else had any luck fixing this? Or am I forced to find another theme?

**Comment by nonepork** [comment]
Same issue, I've learned stoicism along the way of fixing it :p

**Comment by rishikoushikz** [comment]
It was a bug, but I found a simple temporary solution—just disable the snap notifications in Ubuntu settings.

**Comment by LiseAndreasen** [comment]
I don't know whether Orchis or something else is causing this problem. I am simply looking for a workaround. Help.
